app/models/service.rb

```diff
@@ -61,6 +61,8 @@ def self.provider_specific_options(omniauth)
     case omniauth['provider'].to_sym
       when :'37signals'
         { user_id: omniauth['extra']['accounts'][0]['id'], name: omniauth['info']['name'] }
+      when :'lifx'
+        { name: 'LIFX User' }
```

Comment:
Correct. I had a good conversation with the folks at LIFX. They only supply a GUID that represents the user and they feel pretty strongly against sharing either a username or email. There wasn't a uniqueness constraint that I saw about the name, so while this would be confusing if someone had multiple LIFX accounts, it would still work.